cli/utilities/packages.py

```diff
@@ -211,13 +212,96 @@ def unregister(self):
             )
 
     def status(self):
+        """
```

Comment:
This change should not be part of this PR

Reply:
Understood, I’ll remove this change from the PR.